cpp/src/com/google/idea/blaze/cpp/BlazeCWorkspace.java

```diff
@@ -247,6 +292,9 @@ private OCWorkspaceImpl.ModifiableModel calculateConfigurations(
                 .map(file -> "-I" + file.getAbsolutePath())
                 .collect(toImmutableList());
 
+        String rootPath = workspaceRoot.directory().getAbsolutePath();
+        ImmutableList<String> includes = ImmutableList.copyOf(collectIncludes(rootPath, targetKey, blazeProjectData));
```

Comment:
Would you like to take over this pull request? I'm not working on bazel currently and have less knowledge on the plugin source code.
Also I don't think this step is long - it may take time only if enough entries are specified so if one doesn't use this feature at all than it wouldn't cost time. But if you are still worried we may make a progress indicator here.